docs/hugo/content/guide/breaking-changes/breaking-changes-v2.2.0.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "v2.2.0 Breaking Changes"
+linkTitle: "v2.2.0"
+weight: 80
+---
+
+## ManagedClusters ManagedClusterServicePrincipalProfile.Secret field is now marked as a SecretReference
+  We always try to avoid breaking changes, but in this case, allowing raw passwords in the spec is a security problem and as such we've
+  decided to make a break to correct this issue.
+
+**Action required:** If the `ContainerService/ManagedClusters` resource is installed in your cluster and the `ManagedClusterServicePrincipalProfile.Secret` property is set on your `ManagedCluster` resource, below steps should be performed before upgrading:
+
+1. Annotate the resource with `serviceoperator.azure.com/reconcile-policy: skip` to prevent ASO from trying to reconcile the resource while you are upgrading.
+2. Download the current YAML for the resource using `kubectl` if you don't have it elsewhere.
+3. Create a kubernetes secret containing the value for `ManagedClusterServicePrincipalProfile.Secret`.
+4. Edit downloaded YAML in step 2, and add a secret key and name reference. Example [here](https://github.com/Azure/azure-service-operator/blob/main/v2/samples/compute/v1api/v1api20201201_virtualmachine.yaml#L18).
```

Comment:
Can this be done before upgrading, or does this have to happen afterwards?

Reply:
Can be done anytime as at this time, the user is going to update the local copy of their YAML